nixos/modules/services/misc/subsonic.nix

```diff
@@ -37,7 +37,7 @@ let cfg = config.services.subsonic; in {
       };
 
       httpsPort = mkOption {
-        type = types.int;
+        type = types.port;
         default = 0;
```

Comment:
Does this error now?

Reply:
Why would it?

from `lib/types.nix`:
```nix
port = ints.u16;
```
and
```nix
ints.u16 = unsign 16 65536;
```

This means that 0 is included. You can also see this under:
https://search.nixos.org/options?channel=unstable&show=services.httpd.virtualHosts.%3Cname%3E.listen.*.port&from=0&size=50&sort=relevance&query=virtualhost+port

where it says:
> Type    16 bit unsigned integer; between 0 and 65535 (both inclusive)